bundles/org.openhab.binding.emotiva/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="emotiva" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Emotiva Binding</name>
+	<description>This is the binding for devices from the Emotiva Audio Corporation.</description>
+	<connection>local</connection>
+
```

Comment:
Would be nice if this can be extended with suggestion finder details 
https://www.openhab.org/docs/developer/addons/addon.html#discovery-methods